aztec-up/bin/.aztec-run

```diff
@@ -100,7 +100,7 @@ done
 
 # Dynamic port assignment based on IMAGE containing '/aztec' and not containing 'builder' (to exclude aztec-builder)
 port_assignment=""
-if [[ "$IMAGE" == *"/aztec"* ]] && [[ "$IMAGE" != *"builder"* ]]; then
```

Comment:
This if statement seems wild. Shouldn't we just check if AZTEC_PORT has a value, and if so we open the port?

Reply:
Maybe.... It is given a default value at the top of the file. And if aztec-nargo is started with that default (8080) while the sandbox is running at 8080, it blows up

Reply:
checked the commit, `SKIP_PORT_ASSIGNMENT` makes sense to me 👍